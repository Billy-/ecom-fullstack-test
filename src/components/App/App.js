import React from 'react';
import { Router, browserHistory } from 'react-router';

import routes from '../../routes';
import Layout from '../../components/Layout';

import './App.scss';

const App = () => (
    <Layout>
        <Router routes={routes} history={browserHistory}/>
    </Layout>
);

export default App;
