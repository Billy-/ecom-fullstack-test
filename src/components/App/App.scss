@import '../../sass/variables';

body {
  margin: 0;
  font-family: Helvetica, Arial, sans-serif;
  color: $darkGrey;
}
