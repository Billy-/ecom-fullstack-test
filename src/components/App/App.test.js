import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

const props = {};

describe('App', () => {
    it('renders without throwing', () => {
        renderer.create(<App {...props} />);
    });

    it('matches snapshot', () => {
        const tree = renderer
            .create(<App {...props} />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
