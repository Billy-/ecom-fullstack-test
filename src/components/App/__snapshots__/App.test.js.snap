// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
<section
  className="container"
>
  <div
    className="header"
  >
    Header Area
  </div>
  <main>
    <div
      className="container"
    >
      <h2>
        Sorry, the page at "
        blank
        " was not found
      </h2>
    </div>
  </main>
  <div
    className="footer"
  >
    Footer Area
  </div>
</section>
`;
